docker/keycloak/Dockerfile

```diff
@@ -0,0 +1,25 @@
+FROM quay.io/keycloak/keycloak:22.0.1 AS builder
+
+# Enable health and metrics support
+#ENV KC_HEALTH_ENABLED=true
+#ENV KC_METRICS_ENABLED=true
+
+# Configure a database vendor
+ENV KC_DB=mysql
+
+WORKDIR /opt/keycloak
+# for demonstration purposes only, please make sure to use proper certificates in production instead
+RUN keytool -genkeypair -storepass password -storetype PKCS12 -keyalg RSA -keysize 2048 -dname "CN=server" -alias server -ext "SAN:c=DNS:localhost,IP:127.0.0.1" -keystore conf/server.keystore
+RUN /opt/keycloak/bin/kc.sh build
+
+FROM quay.io/keycloak/keycloak:22.0.1
+COPY --from=builder /opt/keycloak/ /opt/keycloak/
+
+# change these values to point to a running postgres instance
+ENV KC_DB=mysql
+#ENV KC_DB_URL=<DBURL>
+#ENV KC_DB_USERNAME=<DBUSERNAME>
+#ENV KC_DB_PASSWORD=<DBPASSWORD>
+#ENV KC_HOSTNAME=localhost
+
+ENTRYPOINT ["/opt/keycloak/bin/kc.sh"]
```

Comment:
Why are we building our own keycloak?

Reply:
It's just a way how I spun `keycloak` in my previous tasks. If we have a reference, how we should use `keycloak`  locally, I'll update this approach

Reply:
Ok.
These docker compose files have keycloak deployments
https://github.com/pm4ml/on-premise-deploy/blob/master/docker-compose/docker-compose.yaml
https://github.com/mojaloop/reporting-hub-bop-shell/blob/master/docker-compose.yaml
Lets try to use the official docker image if possible

Reply:
Updated